transformer_engine/jax/csrc/modules.cpp

```diff
@@ -291,12 +292,17 @@ void LayerNormForwardImpl(size_t n, size_t hidden, void *input, DType in_dtype,
     TensorWrapper dummy_workspace_tensor, dummy_barrier_tensor;
     auto num_sm = cudaDevicePropertiesManager::Instance().GetMultiProcessorCount();
 
+    auto layernorm_fwd_func = zero_centered_gamma ? nvte_layernorm1p_fwd : nvte_layernorm_fwd;
+    if (!is_layer_norm) {
+        assert(!zero_centered_gamma && "rmsnorm doesn't support zero_centered_gamma.");
```

Comment:
We have an assert macro that's more robust and includes more info:
https://github.com/NVIDIA/TransformerEngine/blob/7e1791da0e65ba1684374e4c6add307961ce9b8a/transformer_engine/common/include/transformer_engine/logging.h#L21

```suggestion
        NVTE_CHECK(!zero_centered_gamma, "rmsnorm doesn't support zero_centered_gamma.");
```

Reply:
Sure, we should use the existing marco for it definitely. I have made a commit to revise it